examples/counter/test/sagas.js

```diff
@@ -25,5 +26,20 @@ test('incrementAsync Saga test', (t) => {
     'counter Saga must be done'
   )
 
+  const watcher = rootSaga()
+  const worker = takeEvery('INCREMENT_ASYNC', incrementAsync);
+
+  t.deepEqual(
+    watcher.next().value,
+    worker.next().value,
+    'rootSaga takes from every INCREMENT_ASYNC action'
+  )
+
+  t.equal(
+    watcher.return().done,
```

Comment:
isnt this one testing effectively `true === true`? 

this `watcher.return().done` will always evaluate to `true`